README.md

```diff
@@ -9,73 +16,29 @@ Find some useful links below:
-    - Create a queue manager, queue, and a channel. 
-    - Download [com.ibm.mq.allclient_9.0.5.0_1.0.0.jar](http://central.maven.org/maven2/com/ibm/mq/com.ibm.mq.allclient/9.0.5.0/com.ibm.mq.allclient-9.0.5.0.jar) and [javax.jms-api-2.0.1.jar](http://central.maven.org/maven2/javax/jms/javax.jms-api/2.0.1/javax.jms-api-2.0.1.jar).
-    and copy to the `<SP_HOME>/lib` directory
-                
- 
-- You can use this extension in the latest <a target="_blank" href="https://github.com/wso2/product-sp/releases">WSO2 Stream Processor</a> that is a part of <a target="_blank" href="http://wso2.com/analytics?utm_source=gitanalytics&utm_campaign=gitanalytics_Jul17">WSO2 Analytics</a> offering, with editor, debugger and simulation support. 
-
-- This extension is shipped by default with WSO2 Stream Processor, if you wish to use an alternative version of this extension you can replace the component <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-ibmmq/releases">jar</a> that can be found in the `<STREAM_PROCESSOR_HOME>/lib` directory.
-
-**Using the extension as a <a target="_blank" href="https://siddhi.io/en/v5.0/docs/siddhi-as-a-java-library/">java library</a>**
-
-* This extension can be added as a maven dependency along with other Siddhi dependencies to your project.
-
-```
-     <dependency>
-        <groupId>org.wso2.extension.siddhi.io.ibmmq</groupId>
-        <artifactId>siddhi-io-ibmmq</artifactId>
-        <version>x.x.x</version>
-     </dependency>
-```
-
-## Jenkins Build Status
-
----
-
-|  Branch | Build Status |
-| :------ |:------------ | 
-| master  | [![Build Status](https://wso2.org/jenkins/job/siddhi/job/siddhi-io-ibmmq/badge/icon)](https://wso2.org/jenkins/job/siddhi/job/siddhi-io-ibmmq/) |
-
----
-
 ## Features
 
 * <a target="_blank" href="https://wso2-extensions.github.io/siddhi-io-ibmmq/api/1.1.2/#ibmmq-sink">ibmmq</a> *<a target="_blank" href="https://wso2.github.io/siddhi/documentation/siddhi-4.0/#sink">(Sink)</a>*<br><div style="padding-left: 1em;"><p>IBM MQ sink allows you to publish messages to an IBM MQ broker.</p></div>
 * <a target="_blank" href="https://wso2-extensions.github.io/siddhi-io-ibmmq/api/1.1.2/#ibmmq-source">ibmmq</a> *<a target="_blank" href="https://wso2.github.io/siddhi/documentation/siddhi-4.0/#source">(Source)</a>*<br><div style="padding-left: 1em;"><p>IBM MQ source allows you to subscribe to an IBM message queue and receive messages. It has the ability to receive messages of the 'map' and 'text' message formats.</p></div>
 
-## How to Contribute
- 
-  * Please report issues at <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-ibmmq/issues">GitHub Issue Tracker</a>.
-  
-  * Send your contributions as pull requests to <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-ibmmq/tree/master">master branch</a>. 
- 
-## Contact us 
+## Dependencies
+There are no other dependencies needed for this extension.
```

Comment:
There is a dependency required for IBMMQ extension. Please add details on that.

Reply:
@mohanvive fixed in 015f2bff1fa99c79595063340b1e240eae95c908